### PR TITLE
Provides functionality for unpowered control car

### PR DESCRIPTION
**Pull request by peternewell**
A new feature to facilitate train operation from an unpowered car cab.

https://blueprints.launchpad.net/or/+spec/unpowered-control-car

### CODE REVIEW

File: Source/Documentation/Manual/physics.rst [hunk 1]
```diff
@@ -1963,6 +1963,23 @@ When the auxiliary tender is coupled to the locomotive the *tender* line in the
...
+
+To set a control car up it is necessary to produce an ENG file which has the ``Type ( Control )`` parameter set in 
+the engine section of the ENG file.
+
+The Control car uses most of the same parameters for its configuration as a powered locomotive. The major items that 
+can be left out are those parameters associated with power, motive force, diesel engines, some braking items, such 
+as the compressor and main air reservoir, and some of the diesel effects (as it has no diesel engine).
+
+Some of the cab monitoring gauges provide visibility of what is happening on the powered car. To do this OR searches for 
+the "closest" powered car near the Control car and uses its infromation.
```
Comment (cjakeman) [inline Source/Documentation/Manual/physics.rst:1982]:
```suggestion
the "closest" powered car near the Control car and uses its information.
```

**Review by cjakeman — CHANGES_REQUESTED** [review]
Hi Peter,

This looks good and should add useful functionality.

Please would you complete your Blueprint so I can approve both that and this PR.
A blueprint should explain:
- What functionality is being added
- What the user needs to do (e.g. edit ENG files)
- Does it affect every type of train and every part of the world?
- What parts of the code are affected?
- Is this PR expected to be standalone or the first of a series?

Would you also link your PR to the forum thread.

Thanks,

Chris

File: Source/Documentation/Manual/physics.rst (hunk 1 above)
Comment (peternewell) [reply Source/Documentation/Manual/physics.rst:1982]: Chris,

Thanks for the feedback.

Blueprint updated as suggested.

**Review by cjakeman — APPROVED** [review]
Hi Peter,

Thanks for making those changes.

I do like the comments in your code explaining the conditions in engineering terms.

Approved with thanks and just in time for v1.4.

Chris

**Pull request merged**